floodviz/tests/test_utils.py

```diff
@@ -0,0 +1,416 @@
+import unittest
+import requests_mock
+from floodviz.utils import parse_rdb
+
+
+class TestPeakUtilsRDBparse(unittest.TestCase):
+
+    def setUp(self):
+        self.peak_site = '05481950'
+        self.peak_dv_date = '2008-07-05'
+        self.peak_start_date = '2008-05-20'
+        self.peak_end_date = '2008-07-05'
+        self.endpoint_peak = 'https://nwis.waterdata.usgs.gov/nwis/peak'
+        self.endpoint_dv = 'https://waterservices.usgs.gov/nwis/'
+        self.endpoint_mock = 'http://somethingfake.notadomain/noob/'
+        self.mock_url_peak = 'http://somethingfake.notadomain/noob/?format=rdb&site_no=05481950&start_date='\
+            '2008-05-20&end_date=2008-07-05&agency_cd=USGS'
+        self.mock_url_dv = 'http://somethingfake.notadomain/noob/dv?format=rdb&sites=05481950&startDT='\
+            '2008-07-05&endDT=2008-07-05&siteStatus=all'
+        self.header_values_peak = {'site_no': 0, 'peak_dt': 0, 'peak_va': 0}
+        self.header_values_dv = {'site_no': 0, 'datetime': 0, '43051_00060_00003': 0}
+        self.url_valid_peak = self.endpoint_peak + '?format=rdb' + '&site_no=' + self.peak_site + \
+            '&start_date=' + self.peak_start_date + '&end_date=' + self.peak_end_date + '&agency_cd=USGS'
+        self.url_valid_dv = self.endpoint_dv + 'dv?format=rdb' + '&sites=' + self.peak_site + \
+            '&startDT=' + self.peak_dv_date + '&endDT=' + self.peak_dv_date + '&siteStatus=all'
+        self.test_dict_peak = {
+            'sites': [self.peak_site],
+            'site_query': '&site_no=',
+            'start_date': self.peak_start_date,
+            'start_date_query': '&start_date=',
+            'end_date': self.peak_end_date,
+            'end_date_query': '&end_date=',
+            'format': '?format=rdb',
+            'agency': '&agency_cd=USGS'
+        }
+        self.test_dict_dv = {
+            'sites': [self.peak_site],
+            'site_query': '&sites=',
+            'start_date': self.peak_dv_date,
+            'start_date_query': '&startDT=',
+            'end_date': self.peak_dv_date,
+            'end_date_query': '&endDT=',
+            'format': '?format=rdb',
+            'site_status': '&siteStatus=all',
+            'extra': 'dv'
+        }
+        self.valid_return_peak = [
```

Comment:
It seems like it would be easier if these test cases weren't so long, but it's probably not worth the work to shorten them.